go/teams/get.go

```diff
@@ -40,22 +48,28 @@ func (f *finder) find(ctx context.Context, name string) (*TeamSigChainState, err
 		return nil, err
 	}
 
-	return &state, nil
+	team.Chain = &state
+
+	return team, nil
 }
 
-func (f *finder) chainLinks(ctx context.Context, name string) ([]SCChainLink, error) {
+func (f *finder) rawTeam(ctx context.Context, name string) (*rawTeam, error) {
 	arg := libkb.NewRetryAPIArg("team/get")
 	arg.NetContext = ctx
 	arg.SessionType = libkb.APISessionTypeREQUIRED
 	arg.Args = libkb.HTTPArgs{
 		"name": libkb.S{Val: name},
 	}
-	var chain rawChain
-	if err := f.G().API.GetDecode(arg, &chain); err != nil {
+	var rt rawTeam
```

Comment:
rawTeam has `Box TeamBox`, so the unmarshal fills in TeamBox directly, but I guess `reader_key_masks` isn't in `box` as I thought.